src/Components/test/E2ETest/ServerExecutionTests/ServerReconnectionTest.cs

```diff
@@ -43,6 +45,9 @@ public void ReconnectUI()
             // We should see the 'reconnecting' UI appear
             Browser.Equal("block", () => Browser.FindElement(By.Id("components-reconnect-modal")).GetCssValue("display"));
 
+            new WebDriverWait(Browser, TimeSpan.FromSeconds(30))
```

Comment:
The assertion on line 52 does this check. Did you mean to assert the loader's visibility?

Reply:
This was not my best implementation, but I already fixed it. I wanted to wait for a maximum 30 seconds for the `components-reconnect-modal` to be hidden